File: test_fib.rb
require 'test/unit'
require './fib'

class TestFib < Test::Unit::TestCase
  def test_fib_n_is_0
    assert_equal(1, fib(0))
  end

  def test_fib_n_is_1
    assert_equal(1, fib(1))
  end

  def test_fib_n_is_4
    assert_equal(8, fib(4))
  end

  def test_fib_n_is_10
    assert_equal(89, fib(10))
  end
end

File: tests/test_fib.py
import unittest
import fib

class FibTest(unittest.TestCase):
    def test_fib_n_is_0(self):
        self.assertEqual(fib.fib(0), 1)

    def test_fib_n_is_1(self):
        self.assertEqual(fib.fib(1), 1)

    def test_fib_n_is_4(self):
        self.assertEqual(fib.fib(4), 5)

    def test_fib_n_is_10(self):
        self.assertEqual(fib.fib(10), 89)
